File: domain/src/test/java/com/duxtinto/metroporra/domain/registration/notifyTimeToOpeningInteractorTest.java
package com.duxtinto.metroporra.domain.registration;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class notifyTimeToOpeningInteractorTest {

    private NotifyTimeToOpeningInteractor sutInteractor;

    @BeforeEach
    void setUp() {
        sutInteractor = new NotifyTimeToOpeningInteractor();
    }

    @Test
    void constructor() {
        assertThat(sutInteractor).isInstanceOf(NotifyTimeToOpeningInteractor.class);
    }
}

File: domain/src/test/java/com/duxtinto/metroporra/domain/registration/notifyTimeToOpeningInteractorTest.kt
package com.duxtinto.metroporra.domain.registration

import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

import org.assertj.core.api.Assertions.assertThat

internal class notifyTimeToOpeningInteractorTest {

    private var sutInteractor: NotifyTimeToOpeningInteractor? = null

    @BeforeEach
    fun setUp() {
        sutInteractor = NotifyTimeToOpeningInteractor()
    }

    @Test
    fun constructor() {
        assertThat<NotifyTimeToOpeningInteractor>(sutInteractor).isInstanceOf(NotifyTimeToOpeningInteractor::class)
    }
}
